playbooks/common/openshift-cluster/config.yml

```diff
@@ -42,6 +42,13 @@
     r_openshift_excluder_action: disable
     r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
+- name: Setup Docker
+  hosts: oo_all_hosts
```

Comment:
I don't know how to crack this one offhand. It would be nice to do with one play so maybe we can create a single host group of all containerized hosts or create `oo_X_containerized` groups for containerized hosts and merge them together for this play. We'll know which hosts are containerized by the `openshift.common.is_containerized` fact from `openshift_facts` or just `containerized` from inventory.

* All node hosts.
* Containerized master, etcd, glusterfs, loadbalancer and nfs hosts.